mkdocs.yml

```diff
@@ -1,383 +1,381 @@
+        - Binance Futures: exchanges/binance-perpetual.md
+        - Bybit: exchanges/bybit-perpetual.md
+        - dYdX: exchanges/dydx-perpetual.md
+    - Gateway Connectors:
+      - Gateway Connectors: gateway/index.md
+      - DEXs:
+        - DEXs: gateway/exchanges/index.md
+        - Uniswap: gateway/exchanges/uniswap.md
+        - Pangolin: gateway/exchanges/pangolin.md
+      - Chains:
+        - Chains: gateway/chains/index.md
+        - Ethereum: gateway/chains/ethereum.md
+        - Avalanche: gateway/chains/avalanche.md
+        - Harmony: gateway/chains/harmony.md
+        - Solana: gateway/chains/solana.md
+      - History: gateway/history.md
+    - Global Configs:
+      - Global Configs: global-configs/index.md
+      - Balance Limit: global-configs/balance-limit.md
+      - Color Settings: global-configs/color-settings.md
+      - External Database: global-configs/external-db.md
+      - Kill Switch: global-configs/kill-switch.md
+      - Override Fees: global-configs/override-fees.md
+      - Paper Trade: global-configs/paper-trade.md
+      - Rate Limits Share Pct: global-configs/rate-limits-share-pct.md
+      - Telegram Integration: global-configs/telegram.md
+      - Strategy Autostart: global-configs/strategy-autostart.md
+      - Connect External Database: global-configs/external-db.md
+    - Strategy Configs:
+      - Strategy Configs: strategy-configs/index.md
+      - Add Transaction Costs: strategy-configs/add-transaction-costs.md
+      - Order Levels: strategy-configs/order-levels.md
+      - Filled Order Delay: strategy-configs/filled-order-delay.md
+      - Hanging Orders: strategy-configs/hanging-orders.md
+      - Minimum Spread: strategy-configs/minimum-spread.md
+      - Order Refresh Tolerance: strategy-configs/order-refresh-tolerance.md
+      - Order Optimization: strategy-configs/order-optimization.md
+      - Inventory Skew: strategy-configs/inventory-skew.md
+      - Max Order Age: strategy-configs/max-order-age.md
+      - Order Override: strategy-configs/order-override.md
+      - Price Band: strategy-configs/price-band.md
+      - Price Source: strategy-configs/external-price-source.md
+      - Ping Pong: strategy-configs/ping-pong.md
+      - Rate Oracle: strategy-configs/rate-oracle.md
+    - Troubleshooting: troubleshooting.md
+    - Glossary: glossary.md
+  - Developers:
+      - Developers: developers/index.md
+      - Contribution Guidelines: developers/contributions.md
+      - Architecture: developers/architecture.md
+      - Exchange Connectors:
+        - Developing Connectors: developers/connectors/index.md
+        - Architecture:
+          - Connector Architecture: developers/connectors/architecture/index.md
+          - Order Lifecycle and Market Events: developers/connectors/architecture/order_lifecycle.md
+          - API Throttler: developers/connectors/architecture/api_throttler.md
+        - Build Process: developers/connectors/build.md
+        - QA Testing Process:ndex.md

```

Comment:
Check the output from `mkdocs serve` to detect typos like this
```
WARNING  -  A relative path to 'QA Testing Process:ndex.md' is included in the 'nav' configuration, which is not found in the documentation files
```

I will fix it in an upcoming commit